adam/perception/perception_graph_nodes.py

```diff
@@ -98,6 +98,41 @@ def dot_label(self):
         )
 
 
+@attrs(slots=True, frozen=True, eq=False)
+class Size(GraphNode):
+    x: float = attrib(validator=instance_of(float))
+    y: float = attrib(validator=instance_of(float))
+    area: float = attrib(validator=instance_of(float))
```

Comment:
Is X/Y roughly length, width? The latter may be more clear as variable names.

Reply:
width, height? I'd agree

Reply:
Should I change this in pre-processing as well?

Reply:
That's probably a good idea for consistency's sake.

Reply:
No objections to changing in pre-processing but no strong preference on the rename there.